sql/src/main/java/org/apache/druid/sql/calcite/planner/IngestHandler.java

```diff
@@ -405,7 +405,7 @@ public ExplainAttributes explainAttributes()
     {
       return new ExplainAttributes(
           DruidSqlReplace.OPERATOR.getName(),
-          targetDatasource,
+          targetDatasource.getType(),
```

Comment:
I find it weird that the type is returning the name of the table in tableDestination. Wdyt ? Should we adjust the inteface `IngestDestination` ?

Reply:
This does seem like a mistake.